server/fishtest/views.py

```diff
@@ -594,7 +597,7 @@ def strip_message(m):
         data["base_signature"] = data["new_signature"]
 
     for k, v in data.items():
-        if len(v) == 0:
+        if not k in ['adjudication'] and len(v) == 0:
```

Comment:
`if k not in ['adjudication'] and len(v) == 0:`

Reply:
I guess they are equivalent but `not in` looks indeed nicer.

Reply:
`not in` is an operator, always wondered why not a single word as `notin` or `nin`